sauce/features/general/import-notification/index.js

```diff
@@ -0,0 +1,72 @@
+import { Feature } from 'core/feature';
+// import * as toolkitHelper from 'helpers/toolkit';
+
+export class ImportNotification extends Feature {
+  injectCSS() { return require('./index.css'); }
+
+  constructor() {
+    super();
+    this.isActive = false;
+    this.importClass = 'import-notification';
+    this.invoke = this.invoke.bind(this);
+  }
+
+  willInvoke() {
+    if (this.settings.enabled !== '0') {
+      if (this.settings.enabled === '2') {
+        this.importClass += '-red';
+      }
+
+      // Hook transaction imports so that we can run our stuff when things change
+      ynab.YNABSharedLib.defaultInstance.entityManager._transactionEntityPropertyChanged.addHandler(this.invoke);
+    }
+  }
+
+  shouldInvoke() {
+    return this.settings.enabled !== '0';
+  }
+
+  invoke() {
+    if (!this.isActive) {
+      this.checkImportTransactions();
+    }
+  }
+
+  observe(changedNodes) {
+    if (!this.shouldInvoke()) return;
+
+    // Don't call invoke() if the changed node is a result of us adding or removing
+    // the notification class in checkImportTransactions().
+    if (!changedNodes.has('nav-account-notification') && !this.isActive) {
+      this.invoke();
+    }
+  }
+
+  checkImportTransactions() {
+    this.isActive = true;
+
+    // The following local var is needed because "this" is seen as the currently selected element after the
+    // first $('') type statement.
+    let importClass = this.importClass;
+
+    $('.' + importClass).remove();
+    $('.nav-account-row').each(function (index, row) {
```

Comment:
if you change this function to an arrow function you won't need the handle on `importClass` above

```
$('.nav-account-row').each((index, row) => {
   // ... stuff
});
```